### PR TITLE
Fixed insertion of an id when creating a user.

### PR DESCRIPTION
**Pull request by lmtjanssen**
Fixes bug #85

### CODE REVIEW

File: src/main/java/com/fontys/api/service/UserService.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public UserService(UserRepository userRepository)
     }
 
     @Transactional
-    public User createUser() { return userRepository.save(new User()); }
+    public User createUser(String UUID) { return userRepository.save(new User(UUID)); }
```
Comment (MichielLangen) [inline src/main/java/com/fontys/api/service/UserService.java:22]: UUID has to be unique

**Review by MichielLangen — CHANGES_REQUESTED** [review]

**Review by lfb0801 — DISMISSED** [review]
looks good

File: src/main/java/com/fontys/api/service/UserService.java (hunk 1 above)
Comment (chf3kl9) [reply src/main/java/com/fontys/api/service/UserService.java:22]: hmmm, although you used the 'Unique' notation for the UUID in User.java, it still is possible to create users with the same UUID.

**Review by lfb0801 — APPROVED** [review]
server error isn't ideal, but should be fixed after #84

**Review by MichielLangen — APPROVED** [review]

**Pull request merged**